toolkits/nature/packages/nature-header/scss/50-components/header.scss

```diff
@@ -59,6 +59,10 @@
 	}
 }
 
+.c-header__site-navigation--last {
```

Comment:
Can this not be done without a modifier? E.g. setting padding only if `header__site-navigation + head__site-navigation` or something